### ISSUE TITLE
Error Messages after upgrade from OC 8.04.

### ISSUE THREAD

**Issue by ege1** [owncloud/core#18224]
While upgrading I got this error message:

Updating ownCloud to version 8.1.1, this may take a while.

Wartungsmodus eingeschaltet
Checked database schema update
Checked database schema update for apps
Datenbank aktualisiert
Updated "updater" to 0.6
Updated "files" to 1.1.10
Updated "activity" to 2.0.1
Updated "files_sharing" to 0.6.2
Updated "files_trashbin" to 0.6.3
Updated "files_versions" to 1.0.6
Symfony\Component\Routing\Exception\RouteNotFoundException: Unable to generate a URL for the named route "calendar_index" as such route does not exist.

It looks like the second attempt succeeded, or at least it looks good to me on the first test.

My Version before:
/version.php 
<?php 
$OC_Version = array(8,0,4,2);
$OC_VersionString = '8.0.4';
$OC_Channel = 'stable';
$OC_Build = '2015-06-04T13:45:27+00:00 5fcab24e592c5f0f35d3293449dbd2f15a40f5f8';

Is there something I should have a look at?

Kind Regards,

Erik

**Comment by DeepDiver1975** [comment]
@rperezb yet another upgrade scenario - THX

**Comment by MorrisJobke** [comment]
I had something similiar today with a files a route and an upgrade on Ubuntu using the packages from 8.1.0 to 8.1.1.

**Comment by MorrisJobke** [comment]
Second upgrade attempt just worked fine. (I used the occ tool for both upgrades)

**Comment by rperezb** [comment]
thx, added to the todo list of the upgrade script

**Comment by ege1** [comment]
Is there something wrong with my installation now or went all well finally? Can someone point me to where I need to have a look at?

**Comment by law** [comment]
Ran into this myself, same scenario.  What is the 'occ tool' I can use to help get this upgrade through?

**Comment by nadres** [comment]
Hello, same here when upgrading from 8.1.0 to 8.1.1 via web interface.

`Symfony\Component\Routing\Exception\RouteNotFoundException: Unable to generate a URL for the named route "calendar_index" as such route does not exist.`

I run occ and went OK.

@law you can use `occ` to upgrade from command line. I'm sure it does a lot more than that!
check for `occ` [here](https://doc.owncloud.org/server/8.1/admin_manual/maintenance/upgrade.html).

**Comment by LukasReschke** [comment]
Can reproduce. Awesome… :rocket:

**Comment by LukasReschke** [comment]
cc @karlitschek Breaks a lot.

**Comment by MorrisJobke** [comment]
See also #18315

**Comment by Bugsbane** [comment]
I got this with the web interface upgrade:
> Symfony\Component\Routing\Exception\RouteNotFoundException: Unable to generate a URL for the named route "calendar_index" as such route does not exist.

...but using occ after that worked without error.

Manjaro Linux (Arch spin) OC 8.1.0 > OC 8.1.1

**Comment by MorrisJobke** [comment]
@Bugsbane Yes. The second run always succeeds. This is because of some background loading.

**Comment by nsrosenqvist** [comment]
Happened to me as well when I tried upgrading today. Got it working on second attempt though as previously suggested.

**Comment by fossxplorer** [comment]
On current master, even a second round won't fix the problem.

**Comment by MorrisJobke** [comment]
Fixed in #18839

**Comment by Z4el3m** [comment]
Hi, 
is have a similar problem with upgrade from 7.0.10 to 8.0.8 and also from 7.0.10 directly to 8.1.3 but with contacts app.
I deactivated all apps before (except updater) and upgraded owncloud. 
This worked fine until I activated the "contacts" app again. Since then OC is only a white screen :)
The only interessting thing I found in log after update was this message: 

{"reqId":"5628a3604814e","app":"index","message":"Symfony\\Component\\Routing\\Exception\\RouteNotFoundException: Route \"contacts_index\" does not exist.","level":4,"time":"2015-10-22T08:50:40+00:00","method":"GET","url":"\/owncloud\/index.php\/settings\/ajax\/navigationdetect.php?app=contacts"}

and a lot of:
..."app":"core","message":"Invalid OCS content returned for app 166057","level":4,...

**Comment by coRpTitan** [comment]
I have same issue as Z4el3m mentioned.

./occ upgrade
ownCloud is already latest version

Did not help. I tried like 10 times from web and getting each time same error message. When I enabled older version of calendar I got only blank screen on ANY page of my installation, so I had to disable calendar add-on by using SQL command: 

UPDATE `cloud`.`oc_appconfig` SET `configvalue` = 'no' WHERE `oc_appconfig`.`appid` = 'calendar' AND `oc_appconfig`.`configkey` = 'enabled';

This is what I found at logs(error from calendar):
[Sun Oct 25 14:11:45.584041 2015] [:error] [pid 2932:tid 139957792392960] [client xxx.xxx.xxx.xxx:55189] PHP Fatal error:  Access to undeclared static property: Sabre\\VObject\\Property::$classMap in /.../cloud/apps/calendar/appinfo/app.php on line 43

I hope it will help.
